harness/determined/pytorch/_pytorch_context.py

```diff
@@ -14,7 +14,8 @@
 # Apex is included only for GPU trials.
 try:
     import apex
-except ImportError:
+except ImportError:  # pragma: no cover
+    apex = None
```

Comment:
what is the purpose of `apex = None` here?

Reply:
Ditto on your comment about `amp = None`.

Reply:
ditto